HousingSearchListener/V1/Interfaces/ElasticSearchHelper.cs

```diff
@@ -13,7 +13,17 @@ public ElasticSearchHelper(IElasticClient elasticClient)
             _elasticClient = elasticClient;
         }
 
```

Comment:
I don't think this class should live in the `Interfaces` folder

Reply:
I've created a Helper folder and stored it there